ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/JsonRpcRequest.java

```diff
@@ -31,6 +33,8 @@
 @JsonIgnoreProperties(ignoreUnknown = true)
 public class JsonRpcRequest {
```

Comment:
Can the getParams be removed now that all access is done through the getRequiredParameter and getOptionalParameter methods

Reply:
Not quite - it's still used by a few RPCs - of which I'm not confident to change.